shell/platform/tizen/flutter_tizen_engine.cc

```diff
@@ -129,9 +129,9 @@ bool FlutterTizenEngine::RunEngine() {
       switches.begin(), switches.end(), std::back_inserter(argv),
       [](const std::string& arg) -> const char* { return arg.c_str(); });
 
-  if (std::find(switches.begin(), switches.end(), "verbose-logging") !=
+  if (std::find(switches.begin(), switches.end(), "--verbose-logging") !=
       switches.end()) {
-    SetMinLoggingLevel(DLOG_INFO);
+    Logger::SetLoggingLevel(kLogLevelInfo);
```

Comment:
For the --verbose-logging option, how about using ```kLogLevelDebug```?
Currently, there seems to be no way to print Debug level messages.

Reply:
Actually it's by design: https://github.com/flutter-tizen/engine/pull/75#discussion_r628036964.

You can use `sdb dlog ConsoleMessage` (or an equivalent secure command for TV emulators) to see debug log messages.

Reply:
> Severity | --verbose-system-logs is off (default) | --verbose-system-logs is on
> -- | -- | --
> D | Invisible | Invisible in Flutter Console<br>Visible in Tizen's dlog

> Actually it's by design: #75 (comment).

I already knew this plan. however, logs below the specified log level are ignored.  Therefore, the debug log cannot be printed in both the flutter console and the dlog :(

Reply:
@bbrto21 Ah, you're right. I added a commit which makes debug log messages available through dlog (but not console).